### ISSUE TITLE
Pip installation is missing functionality

### ISSUE THREAD

**Issue by JMMackenzie** [castorini/pyserini#241]
So I've installed pyserini via pip, but the `IndexReader` object has no available `get_term_positions` function:

```
>>> print("\n".join(x for x in dir(index.IndexReader)))
__class__
__delattr__
__dict__
__dir__
__doc__
__eq__
__format__
__ge__
__getattribute__
__gt__
__hash__
__init__
__init_subclass__
__le__
__lt__
__module__
__ne__
__new__
__reduce__
__reduce_ex__
__repr__
__setattr__
__sizeof__
__str__
__subclasshook__
__weakref__
analyze
compute_bm25_term_weight
compute_query_document_score
convert_collection_docid_to_internal_docid
convert_internal_docid_to_collection_docid
doc
doc_by_field
doc_contents
doc_raw
get_document_vector
get_postings_list
get_term_counts
stats
terms
```

I think this is probably some sort of versioning mismatch but should be investigated. Installing via git works fine.

**Comment by lintool** [comment]
hey @JMMackenzie thanks for reporting - digging into this:

```
$ git log tests/test_index_reader.py 
commit 49fd7cb8fd802493dc34f5cb33767d2e72e19f13
Author: zhangyue <nsndimt@gmail.com>
Date:   Thu Aug 6 16:31:15 2020 -0400

    Expose getTermPositions in the IndexReader class from Anserini (#223)
```

The method `get_term_positions` seems to be a feature that post-dates `pyserini-0.9.4.0`, which was released on Jun 26. We're overdue for a new release. Will add this to my TODO.

**Comment by lintool** [comment]
We've published pyserini 0.10.0.0, so this feature should by on PyPI now.
